web/modules/custom/dpl_fees/dpl_fees.module

```diff
@@ -21,6 +21,18 @@ function dpl_fees_dpl_react_apps_data(array &$data): void {
   ];
 }
 
+/**
+ * Implements hook_dpl_protected_nodes_get_protected_nodes().
+ */
+function dpl_fees_dpl_protected_nodes_get_protected_nodes(): array {
+  // Get the configuration form.
+  $form = \Drupal::formBuilder()->getForm('Drupal\dpl_fees\Form\FeesListSettingsForm');
+  // Get the configuration path.
+  $config_path = Url::fromRoute('dpl_fees.settings', [], ['absolute' => TRUE])->toString();
+
+  return dpl_protected_nodes_find_linkit_elements($form, $config_path);
```

Comment:
Wouldn't be an idea to change the parameters of `dpl_protected_nodes_find_linkit_elements()` to `$form_class_path`and `$config_route`. Since you are repeating the same pattern in the `hook_dpl_protected_nodes_get_protected_nodes` hooks?
Then the `dpl_protected_nodes_find_linkit_elements()` function could resolve the context by itself.

Reply:
Just to make sure what you mean, you would like it to be something like this:

`
/**
 * Implements hook_dpl_protected_nodes_get_protected_nodes().
 */
function dpl_fees_dpl_protected_nodes_get_protected_nodes(): array {
  // Get the configuration form.
  $form = \Drupal::formBuilder()->getForm('Drupal\dpl_fees\Form\FeesListSettingsForm');
  // Get the configuration path.
  $config_path = Url::fromRoute('dpl_fees.settings', [], ['absolute' => TRUE])->toString();

  return dpl_protected_nodes_find_linkit_elements('Drupal\dpl_fees\Form\FeesListSettingsForm', 'dpl_fees.settings');
  `

Reply:
I'll leave it as it is for now, as it will take some some time to figure out a better logic when calling the function recursively.

Reply:
Scratch my last comment. I found an easy way to solve this, which I think is quite nice. 
It will also make it possible to add more form/config paths from the same module if needed.